### ISSUE TITLE
Support the new Facebook Beta version

### ISSUE THREAD

**Issue by JanisE** [JanisE/who_is_who_in_socionics#3]
Data loading from the Facebook Beta poll does not work.

**Comment by JanisE** [comment]
This looks rather hopeless to me.

The code of the new site is more dynamic, almost no semantics in the underlying CSS and HTML. There is no plain URL given in the source to load the voter list from. Also, the voter list loading is split into chunks.

But the biggest problem is that I cannot find a sane way to get the "doc_id"'s needed to make the FB API call to get the voter list. It does not match post's ID, and it is even different for the first and the rest of the voter list chunks.

```
const fb_dtsg = document.getElementsByName("fb_dtsg");
$.ajax({url: '/api/graphql', method: 'POST', data: {
fb_dtsg: fb_dtsg, 
variables: '{"optionID":"1367145223479790"}', // input name contains this number
doc_id: '4193882810629373'
}}).done((result) => console.log(result));
```

FB itself gets it from piles of JS code generated and loaded dynamically.

I'm not sure, but a proper solution from the Facebook's point of view might be to create a Facebook App and implement their API within the addon. That is way too complicated and burdensome for the users, as they would probably need to approve the app in the process. And would maybe require even a hosted service from the addon's side.

So, unless someone provides a sane solution to this or some other miracle happens, I guess the functionality of loading poll voters automatically will cease to be along with the old Facebook version.

**Comment by JanisE** [comment]
Found a not too insane way to support poll vote loading from the new Facebook frontend.

Does not (currently) work for polls that does not have at least one option where more than 10 people have voted.

Published in version 3.0.0. The Firefox and Chrome extensions are updated as well.
